Symfony/CS/Fixer/PSR2/ElseifFixer.php

```diff
@@ -30,25 +30,31 @@ class ElseifFixer extends AbstractFixer
     public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
+        for ($index = 7, $count = count($tokens); $index < $count - 1; ++$index) {
```

Comment:
agree to disagree, updated, lets see how the tests do, I guess PHP 7 and/or HHVM will fail